CHANGES.rst

```diff
@@ -35,6 +35,9 @@ Changes in Riot 0.8.17 (2018-10-10)
 MatrixSdk:
  - Upgrade to version 0.9.11.
 
+ Features:
+ - Ability to crop profile picture before setting (#2598)
```

Comment:
Sorry if I wasn't clear. v0.8.17 has been released and will never be modified.
Place move this line to the section 0.8.18 (above), which will be the next release (that's why the date is not set for v0.8.18 yet)

Reply:
i get it now, sry my bad